src/lib.rs

```diff
@@ -13,6 +13,7 @@
 
 extern crate cc;
 extern crate syntex_syntax as syntax;
+extern crate syn;
```

Comment:
Is this perhaps leftover by accident?

Reply:
@alexcrichton yeah ! I was using syn earlier on to convert some `&str`s back to rust types :/